ethcore/types/src/errors/engine_error.rs

```diff
@@ -83,6 +85,7 @@ impl fmt::Display for EngineError {
 			CliqueWrongAuthorCheckpoint(ref oob) => format!("Unexpected checkpoint author: {}", oob),
 			CliqueFaultyRecoveredSigners(ref mis) => format!("Faulty recovered signers {:?}", mis),
 			CliqueTooRecentlySigned(ref address) => format!("The signer: {} has signed a block too recently", address),
+			Config => format!("Configuration error"),
```

Comment:
Not sure if this error adds any information, we could we Custom error with a message instead.